src/imsosorry/uwuification.py

```diff
@@ -107,12 +109,25 @@ def emoji_replace(match: re.Match, strength: float = 0.0) -> str:
     return match_string
 
 
-def uwuify(text: str, *, stutter_strength: float = 0.2, emoji_strength: float = 0.1) -> str:
+def tildes(match: re.Match, strength: float = 0.0):
```

Comment:
All methods need docstrings
```suggestion
def tildes(match: re.Match, strength: float = 0.0):
"""Adds some tildes to spaces."""
```

Reply:
3f65f287a53a96bfca8d1b8700577c730927f688